### DISCUSSION TITLE
Ping showing as '???'

### DISCUSSION THREAD

**Discussion by mateuszkozakiewicz** [yeet-zone/docker-dontstarvetogether#155] [Q&A]
Hi,

I realize this is not strictly related to this docker setup because it is a commonly mentioned issue on Klei forums however people here may be more suited to help.
So basically all the server setup works and people can connect, ping is good etc. but in the server browser the ping is showing up as `???`.
I believe this must be related to the ports setup or the method used for UDP ping. Does anyone have any knowledge how this is done, perhaps this method is commonly filtered/limited on some routers?

**Comment by thasmo** [comment]
@mateuszkozakiewicz, it's probably related to configuring proper port forwarding or related settings in a running firewall or similar. Are you running the server on your local machine or on a dedicated/cloud server?

**Comment by mateuszkozakiewicz** [comment]
It is a machine inside my home network, I have a mikrotik router configured to forward port 10999/UDP however the router is configured for UPnP and it doesn't help.

**Comment by thasmo** [comment]
@mateuszkozakiewicz, I see. It most certainly could also be related to Docker and port configuration/binding itself. If I recall correctly, when I was hosting servers, this was an issue as well, but it somehow started to work in the end - I just can't remember what the issue was or what was needed to fix it.

There might also have been a change in DST, which results in this issue when running the game within Docker containers.

**Comment by mateuszkozakiewicz** [comment]
Same here, used to have this problem with a dedicated server installed on bare metal and the problem went away on its own.. 
No clue what could be the cause, I had the server running for weeks to check if maybe it fixes itself after a time but no luck.
